requirements-dev.txt

```diff
@@ -1,9 +1,10 @@
 check-manifest
 coveralls
 nose
-pdbpp
+pdbpp; os_name == 'posix'
 pyflakes
-pytest
+pytest==3.2.5; python_version == '3.3'
```

Comment:
We might want to split this out into separate version files.
Currently doing research into python build system at work and quite like what nose2 is doing in this regard.
https://github.com/nose-devs/nose2